api/FileReader.json

```diff
@@ -0,0 +1,404 @@
+{
+  "api": {
+    "FileReader": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/FileReader",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": "7"
+          },
+          "chrome_android": {
+            "version_added": "3"
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "3.6",
+            "notes": "Prior to Firefox 4.0 beta 7, all Blob parameters below were File parameters; this has since been updated to match the specification correctly. Prior to Firefox 13.0 the FileReader.error property returned a `FileError` object. This interface has been removed and `FileReader.error` is now returning the `DOMError` object as defined in the latest FileAPI draft."
+          },
+          "firefox_android": {
+            "version_added": "32"
+          },
+          "ie": {
+            "version_added": "10"
+          },
+          "opera": [
+            {
+              "version_added": "11",
+              "notes": "partial"
+            },
+            {
+              "version_added": "12"
+            }
+          ],
+          "opera_android": {
+            "version_added": "11"
+          },
+          "safari": {
+            "version_added": "6"
+          },
+          "safari_ios": {
+            "version_added": "6.1"
```

Comment:
It's weird that Safari for Mac is 6 but Safari for iOS is 6.1, I know this is from the MDN page but I wonder if it's accurate.